.circleci/config.yml

```diff
@@ -72,6 +72,29 @@ jobs:
       - run:
           name: Check black code style
           command: black --check .
+  compile-translations:
+    docker:
+      - image: cimg/python:3.9.16
+    resource_class: small
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Install gettext
+          command: sudo apt-get update && sudo apt-get install gettext
+      - run:
+          name: Enable virtual environment
+          command: echo "source .venv/bin/activate" >> $BASH_ENV
+      - run:
+          name: Compile translation file
+          command: |
+            cd integreat_compass
+            integreat-compass-cli compilemessages --settings=integreat_compass.core.circleci_settings
```

Comment:
I have another question, this time about this line. So this lines refers to the circleci_settings.py file, which this far doesn't exist. However up until now the file mainly just imports the settings.py file. Therefore I would propose to reference the settings file in this line. What do you think?

Reply:
I think we need the dedicated circleci settings file to set a dummy secret key, also it's helpful because we can enable DEBUG logging on CircleCI.

Reply:
Ok, thank you. I already prepared the file like that. But I had a second question about it. Is there a way to circumvent the wildcard import at the beginning? As far as I understand we need all and not just a few dedicated imports from the settings, but also as far as I know this isn't really good practice. So now I'm wondering, what would be best for me to do?

Reply:
Normally, this is bad practice because it imports more than necessary, but right, in this case we want all settings to be present on the custom settings file as well. So there is no possibility to circumvent the wildcard import apart from listing all settings explicitly - but since this are quite a few this doesn't really make sense, also it has the disadvantage that the list needs to be adapted as soon as a setting in the original settings file is added/removed, so in this case the wildcard import is exactly what we want.

Reply:
Ok, thank you for clearing that up